### PR TITLE
Readevents

### PR DESCRIPTION
**Pull request by JonathanPMohan**

### CODE REVIEW

File: src/components/EventItem/EventItem.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import './EventItem.scss';
+
+class EventItem extends React.Component {
+  static propTypes = {
+    event: eventShape.eventShape,
+  }
+
+  render() {
+    const { event } = this.props;
+    console.log(event);
```
Comment (morecallan) [inline src/components/EventItem/EventItem.js]: Remove console logs

File: src/components/EventItem/EventItem.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import eventShape from '../helpers/propz/eventShape';
+
+import './EventItem.scss';
+
+class EventItem extends React.Component {
+  static propTypes = {
```
Comment (morecallan) [inline src/components/EventItem/EventItem.js:7]: Nice with the prop types!

**Review by morecallan — COMMENTED** [review]
Remove console logs, push back up to the branch and I’ll merge!

File: src/components/EventItem/EventItem.js (hunk 1 above)
Comment (JonathanPMohan) [reply src/components/EventItem/EventItem.js]: Alrighty!  I just did that and repushed to the same branch.

**Pull request merged**